controllers/autoneg.go

```diff
@@ -58,19 +58,40 @@ func (e *errNotFound) Error() string {
 // Backend returns a compute.Backend struct specified with a backend group
 // and the embedded AutonegConfig
 func (s AutonegStatus) Backend(name string, port string, group string) compute.Backend {
-	if s.AutonegConfig.BackendServices[port][name].Rate > 0 {
+	cfg := s.AutonegConfig.BackendServices[port][name]
+
+	// Extract initial_capacity setting, if set
+	var capacityScaler float64 = 1
+	if capacity := cfg.InitialCapacity; capacity != nil {
+		// This case should not be possible since validateNewConfig checks
+		// it, but leave the default setting of 100% if capacity is less
+		// than 0 or greater than 100
+		if *capacity >= int32(0) && *capacity <= int32(100) {
+			capacityScaler = float64(*capacity) / 100
+
+			// Special case: GCLB does not accept nonzero scaling factors
+			// less than 10%, so increase these to 10%
+			// See: https://cloud.google.com/load-balancing/docs/backend-service#capacity_scaler
+			if *capacity <= int32(10) {
+				capacityScaler = 0.1
+			}
+		}
+	}
+
+	// Prefer the rate balancing mode if set
+	if cfg.Rate > 0 {
 		return compute.Backend{
 			Group:              group,
 			BalancingMode:      "RATE",
-			MaxRatePerEndpoint: s.AutonegConfig.BackendServices[port][name].Rate,
-			CapacityScaler:     1,
+			MaxRatePerEndpoint: cfg.Rate,
+			CapacityScaler:     capacityScaler,
 		}
 	} else {
 		return compute.Backend{
 			Group:                     group,
 			BalancingMode:             "CONNECTION",
-			MaxConnectionsPerEndpoint: int64(s.AutonegConfig.BackendServices[port][name].Connections),
-			CapacityScaler:            1,
+			MaxConnectionsPerEndpoint: int64(cfg.Connections),
+			CapacityScaler:            capacityScaler,
```

Comment:
I added unit tests to check this but have not yet verified it in an actual cluster